src/api/app/views/webui/users/tokens/index.html.haml

```diff
@@ -0,0 +1,60 @@
+- @pagetitle = 'Tokens'
+
+.card
+  .card-body
+    %h3= @pagetitle
+
+    .text-right
+      = link_to new_token_path do
+        %i.fas.fa-plus-circle.text-primary
+        Create Token
+
+    .text-center= page_entries_info(@tokens)
+
+    - if @tokens.count.positive?
+      .table-responsive
+        %table.table.table-sm.table-striped.table-bordered
+          %thead
+            %tr
+              %th Id
+              %th Operation
+              %th Package
+              %th
+          %tbody
+            - @tokens.each do |token|
+              %tr
+                %td.col-1= token.id
+                %td.col-2= token.class.token_name.capitalize
+                %td.col-auto= link_to("#{truncate(token.package.project.name, length: 80)}/#{truncate(token.package.name, length: 80)}",
+                  package_show_path(project: token.package.project, package: token.package)) if token.package
+                %td.col-1
+                  = link_to '#', title: 'Delete Token', class: 'text-nowrap',
+                    data: { toggle: 'modal', target: '#delete-token-modal', token_id: token.id, action: token_path(token) } do
+                    %i.fas.fa-times-circle.text-danger
+                    Delete
```

Comment:
In tables like in [`Source Files` of a package](https://obs-reviewlab.opensuse.org/eduardoj-webui_for_user_tokens/package/show/home:Admin/ruby), we don't have text for `delete` icon.

```suggestion
```

Reply:
Applied.